be/src/exec/rowid_fetcher.cpp

```diff
@@ -193,6 +193,24 @@ Status RowIDFetcher::_merge_rpc_results(const PMultiGetRequest& request,
     return Status::OK();
 }
 
+bool _has_char_type(const TypeDescriptor& desc) {
+    switch (desc.type) {
+    case TYPE_CHAR:
+        return true;
+    case TYPE_ARRAY:
+    case TYPE_MAP:
+    case TYPE_STRUCT:
```

Comment:
can we make it more general to just check children

Reply:
could not. It has to check the child recursively.....